### ISSUE TITLE
Scrollable place field

### ISSUE THREAD

**Issue by cnjinhao** [cnjinhao/nana#241]
Display scrollbars for a scrollarea field if it contents doesn't fit the area of field.

**Comment by gnobbi** [comment]
hi,
i just like to ask if there is any progress on this feature?

**Comment by cnjinhao** [comment]
No, but I will start working on these new features when I have free time, probably the month after next.

**Comment by gnobbi** [comment]
that would be very nice thank you.
